src/io/flutter/run/test/FlutterTestRunner.java

```diff
@@ -132,10 +133,11 @@ public void processWillTerminate(@NotNull ProcessEvent event, boolean willBeDest
     @Nullable
     @Override
     public String getWebSocketUrl() {
```

Comment:
nit: make this one static helper that gets a wsUrl from an observatoryUri used in both places.

Reply:
👍 will do